bundles/config/org.eclipse.smarthome.config.discovery.usbserial.linuxsysfs/src/main/java/org/eclipse/smarthome/config/discovery/usbserial/linuxsysfs/internal/PollingUsbSerialScanner.java

```diff
@@ -60,9 +62,10 @@
 
     private final Set<UsbSerialDiscoveryListener> discoveryListeners = new CopyOnWriteArraySet<>();
 
-    private final ScheduledExecutorService scheduler = ThreadPoolManager.getScheduledPool(THREAD_POOL_NAME);
-    @Nullable
-    private ScheduledFuture<?> backgroundScanningJob;
+    @NonNullByDefault({})
+    private ScheduledExecutorService scheduler;
```

Comment:
Let's wait for @triller-telekom...

Reply:
See https://github.com/eclipse/smarthome/pull/5839